02week/bio/blog.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.1/css/all.css" integrity="sha384-O8whS3fhG2OnA5Kas0Y9l3cfpmYjapjI0E4theH4iuMD+pLhbf6JI0jIMfYcK3yZ" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="css/blogstyle.css">
+    <meta charset="utf-8">
+    <title>Blog</title>
+  </head>
+  <body>
+    <header id="home">
+      <nav>
+        <a class="active" href="#home"><i class="fas fa-home"></i>  Home</a>
```

Comment:
Is this class being used for anything? I don't see any styling associated with it. If not used, no need to add a class name.

Reply:
Thanks for pointing this out, I meant to use that class and then forgot to add the styling. It's updated now working correctly.